From: Josh Poimboeuf <jpoimboe@redhat.com>
Date: Thu, 9 Jun 2016 14:27:59 +0100
Subject: [PATCH] sched/debug: always show nr_migrations

References: Scheduler performance -- debugging overhead, fate#320486
Patch-mainline: No, expected 4.7 and queued in tip

The nr_migrations field is updated independently of CONFIG_SCHEDSTATS,
so it can be displayed regardless.

Signed-off-by: Josh Poimboeuf <jpoimboe@redhat.com>
Signed-off-by: Mel Gorman <mgorman@suse.de>
---
 kernel/sched/debug.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/sched/debug.c b/kernel/sched/debug.c
index d56c1fe71437..05e5eb9be91f 100644
--- a/kernel/sched/debug.c
+++ b/kernel/sched/debug.c
@@ -566,9 +566,9 @@ void proc_sched_show_task(struct task_struct *p, struct seq_file *m)
 
 	nr_switches = p->nvcsw + p->nivcsw;
 
-#ifdef CONFIG_SCHEDSTATS
 	P(se.nr_migrations);
 
+#ifdef CONFIG_SCHEDSTATS
 	if (schedstat_enabled()) {
 		u64 avg_atom, avg_per_cpu;
 
